paimon-web-ui/src/views/metadata/components/options/index.tsx

```diff
@@ -41,11 +41,12 @@ export default defineComponent({
         key: 'value',
       },
       {
-        title: 'Operation',
+        title: () => <div style={{ paddingRight: '40px' }}>Operation</div>,
         key: 'operation',
+        align: 'right',
```

Comment:
You just need to keep this sentence.

Reply:
I tried align: 'right', but the title just moved to the far right and stuck to the right border, but the action buttons below did not move.

Reply:
this Space component has the `justify` props

Reply:
done,I have used the justify property of Space.

Reply:
If no additional styles are used for the title, there will be no margin with the right border, as shown below：
![image](https://github.com/apache/paimon-webui/assets/34889415/862cb490-496d-4082-bd6c-53e7a22eaeb2)

After adding style：
![image](https://github.com/apache/paimon-webui/assets/34889415/aee14245-8d84-4a62-b81e-154d6748c46d)

Reply:
If you add this padding, the distance from key to the left is not the same as the distance from operation to the right. I think both are fine. In addition, I don't think any users will pay attention to whether the right side is too close, if any, pr welcome.

Reply:
I just think it is unreasonable to choose to write your own style instead of using the relevant props provided by the component.

Reply:
done.